shelley/chain-and-ledger/formal-spec/epoch.tex

```diff
@@ -1288,6 +1288,13 @@ \subsection{Reward Update Calculation}
     \begin{itemize}
       \item Adjust the treasury, reserves and fee pots by the appropriate amounts.
       \item Add each individual reward to the global reward mapping.
+        We must be careful, though, not to give out rewards to accounts
+        that have been deregistered after the reward update was created.
+        \begin{itemize}
+          \item Rewards for accounts that are still registered are added to the reward mappings.
+          \item The sum of the unregistered rewards are added to the reserves.
+            % TODO - We may want to add these to the treasury instead, to be consistent with our other choices.
```

Comment:
Yeah. I added them to the reserves in order to have the same outcome as if they had not been registered in the first place.